### DISCUSSION TITLE
No hamburger menu icon?

### DISCUSSION THREAD

**Discussion by Alwigi** [bootstrap-vue/bootstrap-vue#6455] [Q&A] [answered]
Looking through the icon list I don't see a standalone menu icon, is this done intentionally?

**Comment by Hiws** [comment, marked as answer]
BootstrapVue isn't in charge of the icons. You would have to ask over on the Bootstrap Icons repo.
<http://github.com/twbs/icons/>

**Reply by Alwigi** [reply]
I'll do that, thanks for the reply
